src/ansible_navigator/utils/functions.py

```diff
@@ -443,6 +444,13 @@ def round_half_up(number: Union[float, int]) -> int:
     return int(rounded)
 
 
+def shlex_join(tokens) -> str:
```

Comment:
Maybe good to have type information for `tokens`

Reply:
Also bit of explanation as to why is it handled differently based on python version might help

Reply:
got it